games-mud/tkmoo/tkmoo-0.3.32-r1.ebuild

```diff
@@ -1,44 +1,48 @@
-# Copyright 1999-2016 Gentoo Foundation
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
-inherit eutils
+EAPI=8
 
-MY_PN=${PN/moo/MOO-light}
-MY_P=${P/moo/MOO-light}
+inherit desktop
+
+MY_PN="${PN/moo/MOO-light}"
+MY_P="${P/moo/MOO-light}"
 DESCRIPTION="MOO Client written in Tcl/Tk"
 HOMEPAGE="http://www.awns.com/tkMOO-light/"
 SRC_URI="http://www.awns.com/tkMOO-light/Source/${MY_P}.tar.gz"
+S="${WORKDIR}/${MY_P}"
 
 LICENSE="tkMOO"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE=""
 
 DEPEND="
-	>=dev-lang/tcl-8.3.3:0=
-	>=dev-lang/tk-8.3.3:0="
-RDEPEND=${DEPEND}
-
-S=${WORKDIR}/${MY_P}
+	dev-lang/tcl
+	dev-lang/tk
+"
+RDEPEND="${DEPEND}"
 
 PATCHES=(
 	"${FILESDIR}/${PV}-Makefile-noclean.patch"
-		"${FILESDIR}/${PV}-keys-workaround.patch"
+	"${FILESDIR}/${PV}-keys-workaround.patch"
 )
 
 src_compile() {
-	emake \
-		WISH="$(type -P wish)" \
-		TKMOO_LIB_DIR="/usr/$(get_libdir)/${MY_PN}" \
-		TKMOO_BIN_DIR=/usr/bin
+	local myemakeargs=(
+		WISH="$(type -P wish)"
+		TKMOO_LIB_DIR="${EPREFIX}/usr/$(get_libdir)/${MY_PN}"
```

Comment:
Maybe it'd make more sense to use `${EPREFIX}/usr/share/${PN}`, this is just tcl files. Same for src_install.

No real need to use MY_PN here either, should use the package's gentoo name for its directories if possible.

Reply:
Nice suggestion, I've changed the ebuild and got rid of `MY_PN` completely. :)

Reply:
Still keeping `$(get_libdir)`? I didn't see much point in having tcl script files in lib64, not byte-compiled or anything like that either, it's just arch-independent plain text.

Last thing worth a mention, rest looks good.

Reply:
Completely oversaw this :D 
Fixed now.